wpilibc/src/main/native/include/frc/RobotBase.h

```diff
@@ -33,6 +32,15 @@ void RunRobot(wpi::mutex& m, Robot** robot) {
     theRobot.StartCompetition();
   } catch (const frc::RuntimeError& e) {
     e.Report();
+    FRC_ReportError(
+        err::Error, "{}",
```

Comment:
In Java we report this as a warning rather than an error.  It might be better for it to be an error in both spots, but regardless it seems like we should be consistent.